packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -270,7 +298,7 @@ class _CupertinoSliderRenderObjectWidget extends LeafRenderObjectWidget {
   final int? divisions;
   final Color activeColor;
   final Color thumbColor;
-  final ValueChanged<double>? onChanged;
+  final void Function(double, bool)? onChanged;
```

Comment:
In general function types should always be `typedef`'d before using. Can you search for other `typedef`s for how they're defined?